public/components/Notifications/NotificationForm.tsx

```diff
@@ -168,18 +180,26 @@ export const NotificationForm: React.FC<NotificationFormProps> = ({
                   />
                 </EuiCompressedFormRow>
               </EuiFlexItem>
-              {prepareMessage && (
-                <EuiFlexItem>
-                  <EuiCompressedFormRow>
-                    <EuiSmallButton
-                      fullWidth={false}
-                      onClick={() => prepareMessage(true /* updateMessage */)}
-                    >
-                      Generate message
-                    </EuiSmallButton>
-                  </EuiCompressedFormRow>
-                </EuiFlexItem>
-              )}
+
+              <EuiFlexItem>
+                <EuiCompressedCheckbox
+                  id={`checked`}
+                  label={'Preview message'}
+                  checked={displayPreview}
+                  onChange={(e) => onDisplayPreviewChange(e)}
+                />
+              </EuiFlexItem>
+              {displayPreview ? (
+                <EuiCompressedFormRow label="Message preview" style={{ maxWidth: '100%' }}>
```

Comment:
Can the maxWidth styling be achieved using some OUI props?

Reply:
Done